docs: actually all helm hooks are ignored (#15752)
argoproj/argo-cd: docs/user-guide/helm.md

Unsupported hooks are ignored. In Argo CD, hooks are created by using `kubectl apply`, rather than `kubectl create`. This means that if the hook is named and already exists, it will not change unless you have annotated it with `before-hook-creation`.

!!! warning "Helm hooks + ArgoCD hooks"

>     If you defined Argo CD hooks, All of the Helm hooks will be ignored.   

Reviewer: ```suggestion
    If you define any Argo CD hooks, _all_ Helm hooks will be ignored.   
```
